api/src/model/request/play_asset_pack_fetch_request.gd

```diff
@@ -87,7 +89,10 @@ func get_error() -> PlayAssetPackException:
 # Cancels this fetch request, true if success. 
 # -----------------------------------------------------------------------------
 func cancel_request() -> bool:
-	return PlayAssetPackManager.cancel_asset_pack_request(_pack_name)
+	if _asset_pack_manager == null:
```

Comment:
This logic is only needed for unit testing, since we need to stub the singleton with our mock object.